reactor-core/src/main/java/reactor/core/publisher/SinkManySerialized.java

```diff
@@ -1,9 +1,9 @@
 /*
- * Copyright (c) 2020-Present VMware Inc. or its affiliates, All Rights Reserved.
+ * Copyright (c) 2011-2021 VMware Inc. or its affiliates, All Rights Reserved.
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- *        https://www.apache.org/licenses/LICENSE-2.0
+ *   https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
no empty rows -> this may cause problems with automatic detection for the license